dev-workflow-ui/webContent/view/home.xhtml

```diff
@@ -19,33 +19,31 @@
                 rowKey="#{start.displayName}" rows="10" styleClass="table-header-none"
                 rendered="#{not empty homepageIvyDevWfBean.lastStarts}">
                 <p:ajax event="rowSelect" listener="#{homepageIvyDevWfBean.executeLastStart}" />
-                <p:column styleClass="startedProcessElement">
-                  <div class="flex justify-content-between align-items-center">
```

Comment:
I think the problem was this div. I just moved the classes to the column and it works...
Honestly no idea why 🙈